packages/cozy-harvest-lib/src/stories/AccountSelectorHeader.fixtures.ts

```diff
@@ -0,0 +1,341 @@
+            'bmFjbEyfgDIR8/wLI7G6oicy8llID4Ky+/kz8iBdXWIYT8yQlztQ6bf7PCJrrLgWj4vMsQ5ZuHtJ/pmyfI+7lz3QkS1b6uLK4aCQKSVPqGWimlOg',
+          login: '1 55 55 12 784 125'
+        },
+        cozyMetadata: {
+          createdAt: '2023-03-29T10:25:41.6712839+02:00',
+          createdByApp: 'home',
+          doctypeVersion: '',
+          metadataVersion: 1,
+          updatedAt: '2023-06-23T08:37:37.541Z',
+          updatedByApps: [
+            {
+              date: '2023-06-23T08:37:37.541Z'
+            },
+            {
+              date: '2023-03-29T10:25:41.6712839+02:00',
+              slug: 'home'
+            }
+          ]
+        },
+        defaultFolderPath: '/Administratif/Ameli/1 55 55 12 784 125',
+        identifier: 'login',
+        name: '1 55 55 12 784 125',
+        state: 'LOGIN_SUCCESS'
+      },
+      trigger: {
+        type: '@cron',
+        id: 'cf89ebbc7f71a0aaaa2e7c310f112186',
+        attributes: {
+          _id: 'cf89ebbc7f71a0aaaa2e7c310f112186',
+          _rev: '1-b081bd2dfe95c6594c41678191210f44',
+          domain: 'dev.192-168-1-65.nip.io:8080',
+          prefix: 'cozy7ea0ffaa82da7ef2ba57763982d4f482',
+          type: '@cron',
+          worker: 'konnector',
+          arguments: '0 11 2 * * 3',
+          debounce: '',
+          options: null,
+          message: {
+            account: 'cf89ebbc7f71a0aaaa2e7c310f111c74',
+            konnector: 'ameli',
+            folder_to_save: 'cf89ebbc7f71a0aaaa2e7c310f0fbd76'
+          },
+          current_state: {
+            trigger_id: 'cf89ebbc7f71a0aaaa2e7c310f112186',
+            status: 'done',
+            last_success: '2023-06-23T10:37:40.6638338+02:00',
+            last_successful_job_id: '669b09c00c21de0908c8df614f214cd6',
+            last_execution: '2023-06-23T10:37:40.6638338+02:00',
+            last_executed_job_id: '669b09c00c21de0908c8df614f214cd6',
+            last_failure: '2023-04-26T09:10:39.4642864+02:00',
+            last_failed_job_id: 'de6c1a6ff32a3f11032573ca58008575',
+            last_error: 'VENDOR_DOWN',
+            last_manual_execution: '2023-06-23T10:37:40.6638338+02:00',
+            last_manual_job_id: '669b09c00c21de0908c8df614f214cd6'
+          },
+          cozyMetadata: {
+            doctypeVersion: '1',
+            metadataVersion: 1,
+            createdAt: '2023-03-29T10:25:41.8226664+02:00',
+            createdByApp: 'home',
+            updatedAt: '2023-03-29T10:25:41.8226664+02:00'
+          }
+        },
+        meta: {},
+        links: {
+          self: '/jobs/triggers/cf89ebbc7f71a0aaaa2e7c310f112186'
+        },
+        _id: 'cf89ebbc7f71a0aaaa2e7c310f112186',
+        _rev: '1-b081bd2dfe95c6594c41678191210f44',
+        domain: 'dev.192-168-1-65.nip.io:8080',
+        prefix: 'cozy7ea0ffaa82da7ef2ba57763982d4f482',
+        worker: 'konnector',
+        arguments: '0 11 2 * * 3',
+        debounce: '',
+        options: null,
+        _type: 'io.cozy.triggers'
+      }
+    }
+  ]
+}
```

Comment:
There is no `fixtures` already in our UT?

Reply:
There are some but I didn't found any that were as complete as this (that I took directly from a stack response)